libs/ui/inputs/src/lib/image-input/image-input.component.ts

```diff
@@ -31,13 +30,11 @@ import { FormControl, ReactiveFormsModule } from '@angular/forms'
     FilesDropDirective,
     MatProgressSpinnerModule,
     TranslateModule,
-    ReactiveFormsModule,
   ],
 })
 export class ImageInputComponent {
-  @Input() formControl!: FormControl
   @Input() maxSizeMB: number
-  @Input() previewUrl?: URL
+  @Input() previewUrl?: string
```

Comment:
I think this is the only really necessary change; the type should be switched back to `string`, and the overview-upload component should be adapted accordingly